### PR TITLE
Move references to 1.11

### PR DESCRIPTION
**Pull request by matzew**
/assing @markusthoemmes

### CODE REVIEW

File: templates/csv.yaml [hunk 1]
```diff
@@ -322,7 +322,7 @@ spec:
               containers:
                 - name: knative-openshift
                   # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
-                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-nightly:knative-operator
+                  image: registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.11.0:knative-operator
```
Comment (cardil) [inline templates/csv.yaml:325]: Can't we use variable from project.yaml here?

Comment (matzew) [reply templates/csv.yaml:325]: yep - will do something for that on master

**Comment by matzew** [comment]
Just checked the `registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.11.0:knative-***` images are now there.


/test 4.6-upstream-e2e-aws-ocp-46

**Comment by matzew** [comment]
```
error: some steps failed:
  * could not initialize namespace: could not get push secret: could not read file /etc/push-secret/.dockerconfigjson for secret: open /etc/push-secret/.dockerconfigjson: no such file or directory
time="2020-10-27T16:50:25Z" level=info msg="Reporting job state 'failed' with reason 'initializing_namespace'"
```


/retest

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**